src/state.py

```diff
@@ -16,6 +16,10 @@
 logger = logging.getLogger(__name__)
 
 
+# Bandit false positive
```

Comment:
Perhaps describe in more detail, this doesn't seem to explain why it is safe to ignore beyond that it is being ignored

Reply:
It's a false positive. Bandit thinks that's a password when it's not. I thought the comment "false positive" sufficed

Reply:
What I was thinking is that `Bandit false positive` would be true in any case where we use `no sec`, so it doesn't add more information. If the comment said `Bandit incorrectly thinks this is a password`, then it is clear why we have disabled the rule in this specific case